deploy/signup-with-plugins/signup-with-mock-plugin/delete.sh

```diff
@@ -6,13 +6,13 @@ if [ $# -ge 1 ] ; then
   export KUBECONFIG=$1
 fi
 
-function Deleting_signup_with_plugins() {
+function Deleting_signup_with_mock_plugin() {
   NS=signup
   while true; do
       read -p "Are you sure you want to delete signup helm charts?(Y/n) " yn
```

Comment:
prompt message should reflect clear message which exact plugin is getting deleted.